Targets/Logs/PowerShellConsole.tkape

```diff
@@ -1,17 +1,18 @@
 Description: PowerShell Console Log File
 Author: Mike Cary
-Version: 1.0
+Version: 1.1
 Id: efa4332a-89eb-430c-ab61-006a9e6620d7
 RecreateDirectories: true
 Targets:
     -
         Name: PowerShell Console Log
         Category: PowerShellConsoleLog
         Path: C:\Users\%user%\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadline\
-        FileMask: ConsoleHost_history.txt
+        FileMask: '*_history.txt'
```

Comment:
Yes it works as shown below.

```powershell

PS C:\Users\IEUser\KAPE> ls  C:\Users\IEUSER\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadline\

Directory: C:\Users\IEUSER\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadline

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        8/24/2023   2:54 AM           1441 ConsoleHost_history.txt

PS C:\Users\IEUser\KAPE> .\kape.exe --tsource C: --tdest C:\Users\IEUser\Downloads --tflush --target PowerShellConsole --zip test --gui

# Extract ZIP file using Explorer

PS C:\Users\IEUser> ls .\Downloads\2023-08-24T095629_test\C\Users\IEUser\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadline\ConsoleHost_history.txt

Directory: C:\Users\IEUser\Downloads\2023-08-24T095629_test\C\Users\IEUser\AppData\Roaming\Microsoft\Windows\PowerShell\PSReadline

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        8/24/2023   2:55 AM           1984 ConsoleHost_history.txt
```

The length name is higher because I was using Powershell.

Reply:
Thank you for the due diligence efforts.